### ISSUE TITLE
Control of the laser machine co2

### ISSUE THREAD

**Issue by Revenger72** [arkypita/LaserGRBL#362]
A large request to make the program possible to reassign the "home point" from the lower left corner to the upper left. This is necessary for a CO2 laser, since the laser head is parked in the upper corner.
Also, I wanted to ask the author to make better support for laser co2 control, since when engraving with the laser start command m3, the starting points of the motion are burned, and when command M4 they are not burned because of the higher speed of displacements.
* Well, the desire to add to the program the possibility of sending sequentially and engraving and cutting two different tasks
-------------
 Большая просьба сделать в программе возможность переназначения "точки дом" из нижнего левого угла в верхний левый. Это необходимо для лазера со2, поскольку лазерная головка у него паркуется в верхнем углу.
Так же, хотел попросить автора, сделать лучше поддержку управления лазера со2, т.к. при гравировке с командой включения лазера м3 пережигаются начальные точки движения, а при команде М4 недожигаются из-за большей скорости перемещений.
* Ну и пожелание, добавить в программу возможность отправки последовательно и гравировку и резку двумя разными заданиями

**Comment by brakthehun** [comment]
You can use the $N commands for setting zero in the lower left.
You can up $30 to compensate for the M4 not burning.

**Comment by Revenger72** [comment]
You mean the "mask" of the direction inversion?
I achieved that the head goes to the house in the upper left corner, but the "zero" is obtained in the upper right corner. How to change do not know. The end switches can not be rearranged yet.

**Comment by Revenger72** [comment]
The parameter $30 specifies the maximum laser power. What does the parameter M3 or M4? I do not know English, I translate it into a Google translator. Maybe I did not understand something.

p.s. To avoid overheating (with command M3) on the sides of the image you need to add a gray frame from the sides. But how to make sure that there is no overheating during the engraving of the wind?

**Comment by brakthehun** [comment]
On the final set up,, you can specify a minimum laser power that would correspond to the minimum the laser controller needs to activate.

After your machine does a homing Cycle,, you can set:
$N0=G0Y-(your $131 value)

Then:
$N1=G92X0Y0

**Comment by Revenger72** [comment]
Thank you. This command prescribes in memory where to go to "head" after the endpoints, right? It's not much that I wanted. Or I did not understand.
But I will increase the minimum power. There was a thought, but I decided not to interfere, I thought the machine knows itself as it should.

I need the machine coordinates to be in the upper left corner, like laser machines. But maybe it's not worth puzzling, I'll soon get used to it. They reset the coordinates of the endpoints, then moved the "head" to the desired point and then reset the "working" coordinates.
